front/app/components/FilterSelector/Combobox.tsx

```diff
@@ -137,11 +137,14 @@ const Combobox = ({
             minWidth={minWidth ? minWidth : undefined}
             onKeyDown={handleKeyDown}
             ariaExpanded={opened}
-            aria-controls={baseID}
+            ariaControls={baseID}
             // Needed to track aria-labelledby
             id={`${baseID}-label`}
             role="combobox"
             aria-haspopup="listbox"
+            ariaLabel={
+              typeof currentTitle === 'string' ? currentTitle : undefined
+            }
```

Comment:
Would it be better to have an `ariaLabel` prop as well for when the `currentTitle` is a JSX element? :thinking:

Reply:
I agree but couldn't figure out how to do it easily. Do you have an idea?

Reply:
Oh my idea was more relying on the developer to pass an `ariaLabel` prop if they're using a `JSX.Element` for the `currentTitle`. Agree that finding a way to automatically do this in the code would be challening :thinking: I'm not sure myself how that could be done.

Reply:
This will probably be flagged when we add those accessibility checks even more and then we can resolve based on the use case 👍